client/src/components/annotators/ImageAnnotator.vue

```diff
@@ -156,13 +163,13 @@ export default {
       return new Promise((resolve) => {
         const img = new Image();
         img.crossOrigin = 'Anonymous';
```

Comment:
Should this also be included in the default function?

Reply:
Had to look this up, because I forgot what it does.  It doesn't affect how an image is fetched.   It says "If an image is from another origin, and that origin specifies an adequate matching CORS setting, then allow the image to be used on a canvas".

So no, all images regardless of where/how they're loaded can have this.